pdf-search/src/main/scala/Main.scala

```diff
@@ -0,0 +1,24 @@
+import java.io.File
+
+import org.apache.commons.lang3.StringUtils
+import org.apache.pdfbox.pdmodel.PDDocument
+import org.apache.pdfbox.text.PDFTextStripper
+
+object Main {
+  def main(args: Array[String]): Unit = {
+    println(pdfContainsPhrase("D:/tmp/KeepTalkingAndNobodyExplodes-BombDefusalManual-v2-pl.pdf", "bomb"))
```

Comment:
could you replace this path with something like `getClass.getResource(path)`?

Reply:
Done